Kinvey/Kinvey/MIC.swift

```diff
@@ -475,7 +475,7 @@ class MICLoginViewController: UIViewController, WKNavigationDelegate, UIWebViewD
             let object = try? JSONSerialization.jsonObject(with: data),
             let json = object as? JsonDictionary
         {
-            failure(error: Error.unknownJsonError(httpResponse: nil, data: nil, json: json))
+            failure(error: Error.unknownJsonError(httpResponse: nil, data: data, json: json))
```

Comment:
Why was this changed?

Reply:
sorry, change not related to delta set. I will remove that.